tools/db_bench_tool.cc

```diff
@@ -278,7 +278,34 @@ DEFINE_double(read_random_exp_range, 0.0,
               "num * exp(-r) where r is uniform number from 0 to this value. "
               "The larger the number is, the more skewed the reads are. "
               "Only used in readrandom and multireadrandom benchmarks.");
-
+DEFINE_double(key_dist_a, 0.0,
+              "The parameter 'a' of key access distribution model "
+              "f(x)=a*exp(b*x)+c*exp(d*x)");
```

Comment:
Do we need both of `a` and `c`?

Reply:
Users can only input a or c, then, it will regress to the regular exponential model. As tested in the fitting model, if we only fit the regular exponential model, the error can be large especially when the access count is between 3-20.